chunks/modelevaluation.ipynb

```diff
@@ -0,0 +1,935 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Evaluating Models\n",
+    "\n",
+    "Goals:\n",
+    "\n",
+    "* Be able to design and carry out a posterior predictive model check, and describe its relationship to a frequentist significance test\n",
+    "\n",
+    "* Recognize the Bayesian Evidence, know how it is computed, and describe the difference between a Bayes Factor and a likelihood ratio "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Further reading\n",
+    "\n",
+    "* Mackay, Chapters 3 and 28\n",
+    "\n",
+    "* Gelman et al, Chapters 6 and 7"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Evaluation\n",
+    "\n",
+    "* You can't do inference without making assumptions.\n",
+    "\n",
+    "* We must _test_ the hypotheses defined by our models.\n",
+    "\n",
+    "\n",
+    ">  Does the model provide an _accurate_ description of the data?\n",
+    ">  \n",
+    ">\n",
+    ">  Are there alternative models that are either more accurate, more efficient, or both?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Checking\n",
+    "\n",
+    "* How do we know if our model is any good? One property that \"good\" models have is *accuracy.*\n",
+    "\n",
+    "\n",
+    "* Accurate models generate data that is *like* the observed data. What does this mean? \n",
+    "\n",
+    "\n",
+    ">  * A model's ability to \"predict the data\" can often be assessed _visually_\n",
+    "> \n",
+    ">\n",
+    ">  * *Test statistics* can be designed to capture discrepancies between observed and predicted data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Model Checking Example\n",
+    "\n",
+    "We'll step through a simple [example analysis](../examples/Straightline), of some $(x, y, \\sigma)$ data drawn generated by a simple linear model:\n",
```

Comment:
1. "drawn generated"
2. "linear model" is (colloquially) misleading. I imagine that's the point, but they are so obviously not on a line
3. sigma kind of implies that we will fit for an intrinsic scatter also, so maybe that can be replaced by "error" or something